Kmozart/free-mentors
# #168024208 Sets up Eslint

#### What does this PR do?
- Have Eslint in API set up.

#### Description of Task to be completed?
- Configuring API with the required standards and eslint.

#### How should this be manually tested?
- ```git clone``` this repo and in the terminal, check for the eslint version by typing ```eslint -v```.  Also, check for the ``.eslintrc`` configured file.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- #168024208 ch-setup-lint

#### Screenshots (if appropriate)
-------------------------------------------------------------------------------------------------------------
 ![eslint setup](https://user-images.githubusercontent.com/48585401/63457699-25b17b80-c45a-11e9-9f98-5d8a4d09e0fd.PNG)

#### Questions:
- N/A
